release/artifacts.yaml

```diff
@@ -99,11 +99,27 @@
   type: jar-scala
 - target: //ledger/ledger-api-health:ledger-api-health
   type: jar-scala
-- target: //ledger/ledger-api-test-tool:ledger-api-test-tool
+- target: //ledger/ledger-api-tests/infrastructure:infrastructure-1.14
```

Comment:
@gerolf-da: I have added a bunch of new Maven release targets as a result of this change. Would you mind reviewing them and deciding if they're appropriate, or suggest someone who can?

Reply:
@moritzkiefer-da, @garyverhaegen-da, care to review?

Reply:
I don't think I'm the right person to review this. I don't even know enough about it to know who to turn to. All I can say is the YAML syntax seems fine to me.

This file controls what we upload to Maven Central, which means it is technically part of our public API. I _think_ we don't consider renaming artifacts a "very" breaking change, as long as all of the functionality is still covered and all people need to do is rename their dependencies, but this should definitely be mentioned in the changelog (which it may already be, I haven't checked).

Reply:
The existing Maven artifacts haven't changed; the Bazel targets have. So the public API should be identical.